lib/graphql/batch/mutation_execution_strategy.rb

```diff
@@ -1,18 +1,18 @@
 module GraphQL::Batch
-  class MutationExecutionStrategy < GraphQL::Query::SerialExecution
-    class FieldResolution < GraphQL::Query::SerialExecution::FieldResolution
+  class MutationExecutionStrategy < GraphQL::Batch::ExecutionStrategy
+    class FieldResolution < GraphQL::Batch::ExecutionStrategy::FieldResolution
       def get_finished_value(raw_value)
+        return super if execution_context.strategy.disable_batching
+
         raw_value = GraphQL::Batch::Promise.resolve(raw_value).sync
 
-        context = execution_context.query.context
-        old_execution_strategy = context.execution_strategy
+        execution_context.strategy.disable_batching = true
```

Comment:
looks like https://github.com/rmosolgo/graphql-ruby/pull/379 changed in https://github.com/rmosolgo/graphql-ruby/pull/379 so this no longer works in graphql-ruby now that 1.2 has been released (https://travis-ci.org/Shopify/graphql-batch/jobs/174014089).

GraphQL::BatchTest#test_mutation_batch_subselection_execution:
NoMethodError: undefined method `strategy' for #<GraphQL::Query::Context::FieldResolutionContext:0x00000002e23180>
    /home/travis/build/Shopify/graphql-batch/lib/graphql/batch/mutation_execution_strategy.rb:5:in `get_finished_value'